src/main/java/org/gridsuite/boundary/importer/job/CgmesBoundaryServiceRequester.java

```diff
@@ -0,0 +1,120 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.boundary.importer.job;
+
+import org.json.JSONArray;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.math.BigInteger;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.nio.charset.StandardCharsets;
+import java.time.Duration;
+import java.util.*;
+
+/**
+ * @author Franck Lecuyer <franck.lecuyer at rte-france.com>
+ */
+public class CgmesBoundaryServiceRequester {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(CgmesBoundaryServiceRequester.class);
+
+    private static final String API_VERSION = "v1";
+
+    private final String serviceUrl;
+
+    private final HttpClient httpClient;
+
+    public CgmesBoundaryServiceRequester(String serviceUrl) {
+        this.serviceUrl = serviceUrl;
+        httpClient = HttpClient.newHttpClient();
+    }
+
+    public boolean importBoundary(TransferableFile boundaryFile) throws IOException, InterruptedException {
+        Map<Object, Object> data = new LinkedHashMap<>();
+        data.put("file", boundaryFile);
+
+        // Random 256 length string is used as multipart boundary
+        String boundary = new BigInteger(256, new Random()).toString();
+
+        HttpRequest request = HttpRequest.newBuilder()
+                .uri(URI.create(serviceUrl + API_VERSION + "/boundaries"))
+                .timeout(Duration.ofMinutes(2))
+                .header("Content-Type", "multipart/form-data;boundary=" + boundary)
+                .POST(ofMimeMultipartData(data, boundary))
+                .build();
+
+        HttpResponse<String> response = httpClient.send(request, HttpResponse.BodyHandlers.ofString());
+        return response.statusCode() == 200;
```

Comment:
20x ?

Reply:
Same test is done in cgmes-assembling-job and case-import-job
And 200 is the standard HTTP response code when everything is OK